python/foglamp/services/south/server.py

```diff
@@ -199,6 +212,15 @@ def run(self):
         # This activates event loop and starts fetching events to the microservice server instance
         loop.run_forever()
 
+    async def _stop_and_restart(self):
+        try:
+            self._plugin._plugin_stop(self._plugin_handle)
```

Comment:
there is no standard `_plugin_stop`, plugin contract is never supposed to have this method!